### ISSUE TITLE
[bug] User details modification throws error

### ISSUE THREAD

**Issue by vovpff** [YetiForceCompany/YetiForceCRM#14209]
# 🐞 bug report

### ⁉️ Describe the bug

After clicking Save at user details edit page there are appears `Error occurs, please inform administrator` error message and endless loading.

All user modifications applied and saver but page stuck on this.

I tried this on gitdev but can't reproduce this.   

### 📷 Screenshot of configuration

![ConfReport](https://user-images.githubusercontent.com/8545791/107185307-4c1efa00-69f3-11eb-8d98-e7b07e4cd60a.png)

### 📝 PHP/Apache/Nginx/Browser/CRM Logs

 cache/logs/system.log

```
2021-02-08 09:23:12.9692 [info][yii\db\Connection::open] - Opening DB connection: mysql:host=localhost;dbname=crm;port=3306
2021-02-08 09:23:12.97 [info][yii\db\Command::query] - SELECT EXISTS(SELECT * FROM `vtiger_users` WHERE (`id`=1) AND ((`status`='Active') AND (`deleted`=0)))
2021-02-08 09:23:12.9725 [info][yii\db\Command::query] - SELECT `name` FROM `a_yf_settings_modules` WHERE `status`=1
2021-02-08 09:23:12.9741 [info][yii\db\Command::query] - SELECT * FROM `vtiger_tab`
2021-02-08 09:23:12.9748 [info][yii\db\Command::query] - SELECT * FROM `vtiger_entityname`
2021-02-08 09:23:12.9762 [error] - Call to a member function getId() on null
Error: Call to a member function getId() on null in crm_root/cache/templates_c/basic/728e07011854a444313089da304a74b8804fbfa7_0.file.UserViewHeader.tpl.php:24
Stack trace:
#0 crm_root/vendor/smarty/smarty/libs/sysplugins/smarty_template_resource_base.php(123): content_6020d7e0d7c107_18050191(Object(Smarty_Internal_Template))
#1 crm_root/vendor/smarty/smarty/libs/sysplugins/smarty_template_compiled.php(114): Smarty_Template_Resource_Base->getRenderedTemplateCode(Object(Smarty_Internal_Template))
#2 crm_root/vendor/smarty/smarty/libs/sysplugins/smarty_internal_template.php(216): Smarty_Template_Compiled->render(Object(Smarty_Internal_Template))
#3 crm_root/vendor/smarty/smarty/libs/sysplugins/smarty_internal_templatebase.php(232): Smarty_Internal_Template->render(false, 1)
#4 crm_root/vendor/smarty/smarty/libs/sysplugins/smarty_internal_templatebase.php(134): Smarty_Internal_TemplateBase->_execute(Object(Smarty_Internal_Template), NULL, NULL, NULL, 1)
#5 crm_root/include/runtime/Viewer.php(207): Smarty_Internal_TemplateBase->display('modules/Users/U...')
#6 crm_root/modules/Settings/Users/views/Detail.php(75): Vtiger_Viewer->view('UserViewHeader....', 'Users')
#7 crm_root/include/main/WebUI.php(189): Settings_Users_Detail_View->process(Object(App\Request))
#8 crm_root/index.php(35): Vtiger_WebUI->process(Object(App\Request))
#9 crm_root/public_html/index.php(11): require('/home/crm/e...')
#10 {main}
```

### 🌍 Your Environment

| Environment                         | Version / Name |
| ----------------------------------- | -------------- |
| YetiForce                           | 6.1

**Comment by rskrzypczak** [comment]
Could you be more specific?
What have you changed?
Have you tested it on https://gitstable.yetiforce.com or https://gitdeveloper.yetiforce.com ?

**Comment by vovpff** [comment]
I has tested on gitstable and gitdev and can't reproduce this.
Also I has tuned webserver and now only https is red but this tuning doesn't affect to problem.
Other thing what I do: I has put {print_r($RECORD, true)} and {print_r($USER_MODEL, true)} to UserViewHeader.tpl and noticed what this variables not null before saving.

**Comment by vovpff** [comment]
@rskrzypczak Many thanks! This works!

**Comment by mariuszkrzaczkowski** [comment]
Fixed in 6.1.0_ServicePack_v7

**Comment by vovpff** [comment]
@mariuszkrzaczkowski Thank you!
